pkg/providers/vsphere/vmlifecycle/bootstrap_linuxprep_test.go

```diff
@@ -84,7 +85,7 @@ var _ = Describe("LinuxPrep Bootstrap", func() {
 			}
 
 			vmCtx = pkgctx.VirtualMachineContext{
-				Context: context.Background(),
+				Context: pkgcfg.NewContextWithDefaultConfig(),
```

Comment:
This can have a dramatic change in effect on tests if the contents of the default config changes later. Setting it up explicitly probably makes more sense like the current approach uses.